src/store/getters.ts

```diff
@@ -9,3 +32,355 @@ export const getNameRequestUrl = (state: any): string => {
 export const isAuthenticated = (state: any): boolean => {
   return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
 }
+
+/** Returns the name. */
+export const getName = (state: StateIF): string => {
+  return state.stateModel.newRequestModel.name
+}
+
+/** Returns the original name. */
+export const getOriginalName = (state: StateIF): string => {
+  return state.stateModel.newRequestModel.nameOriginal
+}
+
+/** Returns the assumed name. */
+export const getAssumedName = (state: StateIF): string => {
+  return state.stateModel.newRequestModel.assumedNameOriginal
+}
+
+/** Returns the nr. */
+export const getNr = (state: StateIF): Partial<NameRequestI> => {
+  return state.stateModel.newRequestModel.nr
+}
+
+/** Returns the nr id. */
+export const getNrId = (state: StateIF): string => {
+  return state.stateModel.newRequestModel.nr.id
```

Comment:
OK, you didn't use optional chaining. I think that's very good for now as it won't hide errors. We'll have to be careful where we get NR id, NR state, etc to check that we have a valid NR first.

Reply:
> We'll have to be careful where we get NR id, NR state, etc to check that we have a valid NR first.

... either with an explicit check, or better, the subject code doesn't get called if there is no NR in store.

Reply:
You've read my mind, i recalled an earlier comment you made about hiding errors, which we didn't want quite yet.